fixtures/app.data.ts

```diff
@@ -52,6 +52,7 @@ const appData = {
         children: [USERS_LINK, CONTRIBUTORS_LINK],
       },
     ],
+    account: { label: "account.manage", children: [] },
```

Comment:
Not sure if this is the correct label for this placeholder.

Reply:
What should the label be? If it's "Manage", then use "header.manage". If it's "Account", then use "header.account". The key/values are here: https://github.com/NGLPteam/wdp-admin-ui/blob/main/locales/en.json#L144-L148
You can always add new translations to `en.json` if you need to!